### PR TITLE
feat: dynamic grid on new collection

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).

👇 \_ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Context

- [x] Related #4444

#### Before submitting pull request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

#### Optional

- [ ] I've tested it at </bsx/collection>
- [ ] I've tested PR on mobile
- [ ] I've written unit tests 🧪
- [ ] I've found edge cases

#### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)

#### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

## Screenshot 📸

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

https://user-images.githubusercontent.com/734428/221822306-1ba1c6b1-2582-4861-a92c-4cc8998e802b.mov

### CODE REVIEW

File: components/shared/DynamicGrid.vue [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  }
+)
+
+const gridCols = computed(() => {
+  return {
+    display: 'grid',
+    gap: '2rem',
+    gridTemplateColumns: `repeat(${cols.value}, minmax(0, 1fr))`,
+  }
+})
```
Comment (roiLeo) [inline components/shared/DynamicGrid.vue:48]:
```suggestion
const gridCols = computed(() => ({
    display: 'grid',
    gap: '2rem',
    gridTemplateColumns: `repeat(${cols.value}, minmax(0, 1fr))`,
}))
```

**Review by roiLeo — CHANGES_REQUESTED** [review]
I would prefer not to add the size of the layout in query since this is user preference.
store the grid changes in vuex `preferences` for now

**Review by roiLeo — APPROVED** [review]
Noice!
✅ wfm

**Review by Jarsen136 — APPROVED** [review]

**Comment by exezbcz** [comment]
@preschian And it is also applied to explore as shown in the screen record? Or does this apply only to collection? So far its looking good!

**Comment by roiLeo** [comment]
🚀

**Pull request merged**